devfile.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: 1.0.0
+metadata:
+  generateName: che-theia-
+
+components:
+  - mountSources: true
+    command:
+      - tail
+      - '-f'
+      - /dev/null
+    memoryLimit: 512Gi
+    type: dockerimage
+    image: 'quay.io/eclipse/che-theia:next'
+    alias: che-theia-next-dev
+    env:
+      - value: 0.0.0.0
+        name: THEIA_HOST
+      - value: '3130'
+        name: THEIA_PORT
+      - value: '0'
+        name: NODE_TLS_REJECT_UNAUTHORIZED
+
+  - mountSources: true
+    memoryLimit: 3Gi
+    type: dockerimage
+    image: 'quay.io/eclipse/che-theia-dev:next'
+    alias: che-dev
+
+  - id: redhat/vscode-yaml/latest
+    type: chePlugin
+
+  - id: che-incubator/typescript/latest
+    memoryLimit: 2048M
+    type: chePlugin
+
+  - id: github/vscode-pull-request-github/latest
+    type: chePlugin
+
+commands:
+
+  - name: build ... current yarn project
+    actions:
+      - workdir: '${fileDirname}'
+        type: exec
+        command: >
+          echo "build from ${fileDirname}" && killall node; yarn || (yarn
```

Comment:
yarn || (yarn lint:fix && yarn format:fix && yarn) 

not sure it's what people want ? it will run twice on some cases where there is no lint/format error but only compilation error

Reply:
right, what about
```
yarn --ignore-scripts && yarn lint:fix && yarn format:fix && yarn
```